### PR TITLE
Update config docs

### LINKED ISSUE #90: update config page with WAL logging syntax
See https://github.com/influxdb/influxdb.com/issues/550.

Apparently in 0.9.5 the syntax is different than 0.9.6, although I can't explain why from looking at the code.

### LINKED ISSUE #112: minor new config setting
https://github.com/influxdb/influxdb/blob/a63d2623ffa565335e953f01af78120e92e0358c/etc/config.sample.toml#L335

### PR DESCRIPTION
**Pull request by gunnaraasen**
Fixes #90 

Also, fixes #112

### CODE REVIEW

File: content/influxdb/v0.9/administration/config.md [hunk 1]
```diff
@@ -6,303 +6,377 @@ menu:
...
 ---
 
-## The basics
----
-Generate a new configuration file by running `influxd config` and redirecting the output to a file:
+The InfluxDB configuration file contains configuration settings specific to a local node.
+
+## Using Configuration Files
+
+The `influxd config` command will print out a new TOML-formatted configuration with all the available configuration options set to their default values. On unix systems, a new configuration file can be generated by redirecting the output of the command to a file.
```
Comment (beckettsean) [inline content/influxdb/v0.9/administration/config.md]: perhaps "on POSIX systems"

File: content/influxdb/v0.9/administration/config.md [hunk 2]
```diff
@@ -6,303 +6,377 @@ menu:
...
+
 ```
-influxd config  > /etc/influxdb/influx_v0.9.x.conf
+influxd config > /etc/influxdb/influxdb.conf.generated
 ```
 
-Start the influxd process using that configuration file with `influxd -config`:  
+Generally, configuration files from prior versions of InfluxDB 0.9 should work with the latest 0.9 release. However, occasionally configuration options are changed or added. A new configuration file should be generated every time InfluxDB is upgrade or settings from an old configuration file can be preserved by running the `influxd config` command with the `-config` option.
```
Comment (beckettsean) [inline content/influxdb/v0.9/administration/config.md]: upgrade**d**.

I'm confused by "or settings from an old configuration file can be preserved by running the `influxd config` command with the `-config` option." What are you trying to recommend?

**Comment by beckettsean** [comment]
A few notes, but it's hard to read the changelist. A lot of things show as changed that are really just moved a bit in the doc. I did not do a thorough read.

**Comment by gunnaraasen** [comment]
Made some updates. Not sure if there's a way to reduce the diffs from moving things around.

**Pull request merged**